testutil/keeper/tokenomics.go

```diff
@@ -360,6 +374,11 @@ func NewTokenomicsModuleKeepers(
 	)
 	require.NoError(t, sharedKeeper.SetParams(sdkCtx, sharedtypes.DefaultParams()))
 
+	if params, ok := cfg.moduleParams[sharedtypes.ModuleName]; ok {
```

Comment:
Ensure each module's params are updated which is present in the map (not just the shared module).